ansi/sixel.go

```diff
@@ -0,0 +1,73 @@
+package ansi
+
+import (
+	"bytes"
+	"strconv"
+)
+
+type SixelSequence struct {
```

Comment:
Sixel is just a DCS command. This should be a `type SixelImage struct` that conforms to `image.Image`.

Reply:
Sixels don't really lend themselves to random access by pixel- Image.At() would basically have to be O(N) where N is the number of bytes in the encoded sixel image.